site/types/index.js

```diff
@@ -1,5 +1,7 @@
 import type { Tweet } from './tweet';
 import type { InstagramPost } from './instagramPost';
+import type { Badger } from './badger';
 
 export { Tweet };
 export { InstagramPost };
+export { Badger };
```

Comment:
`export {Tweet, InstagramPost, Badger}` or we can use the draft support for re-exporting: http://jamesknelson.com/re-exporting-es6-modules/.